munit-scalacheck/shared/src/main/scala/munit/scalacheck/ScalaCheckSuite.scala

```diff
@@ -0,0 +1,39 @@
+package munit
+package scalacheck
+
+import org.scalacheck.Prop
+import org.scalacheck.{Test => ScalaCheckTest}
+import org.scalacheck.util.Pretty
+import scala.concurrent.Future
+
+abstract class ScalaCheckSuite extends FunSuite {
```

Comment:
I wonder if this should be a trait that users mix instead? It would be nice if users could mix in multiple helpers (cats integration, scalacheck, ...)